packages/react-jss/src/createHoc.js

```diff
@@ -32,7 +34,19 @@ const dynamicStylesNs = Math.random()
  *
  */
 
-const getStyles = (stylesOrCreator, theme) => {
+type Props = {
+  classes: ?{},
+  innerRef?: () => {},
```

Comment:
should be `(comp: ComponentType<{}> | null) => void`. You could make the component type more exact with moving the props into the generic types of `createHOC`

Reply:
Used `(comp: ComponentType<{}> | null) => void` in the current implementation, will implement generics in the next PR, it's still a bit of overhead for me still